buildozer/__init__.py

```diff
@@ -508,46 +505,6 @@ def _install_application_requirement(self, module):
                  env=self.env_venv,
                  cwd=self.buildozer_dir)
 
-    def check_garden_requirements(self):
-        '''Ensure required garden packages are available to be included.
-        '''
-        garden_requirements = self.config.getlist('app',
-                'garden_requirements', '')
-
```

Comment:
Maybe we should emit some kind of `removed warning` (or maybe raise an exception) in case that we found `garden_requirements` inside a `buildozer.spec` file. This way, users could move the garden requirements...otherwise all garden packages will be silently omitted, am I right?

Reply:
Addressed (but untested)